ner_processing/master_mapping.py

```diff
@@ -0,0 +1,180 @@
+"""
+This file specifes the CAN and data ID mappings. There are three levels of IDs specified:
+    - External Message ID (actual CAN message id)
+    - Internal Message ID (internal id for messages to allow easier changing of external ids)
+    - Data ID (id for individual data values contained in the messages)
+"""
+
+from ner_processing.decode_data import *
+
+# Mapping from external to internal message IDs
+MESSAGE_IDS = {
```

Comment:
Is this a way of abstracting the CAN messages into easy-to-remember data structures, or will this doc have to be used as a key while developing processing code? Also what would be the method for adding/deleting CAN messages that we want to process?

Reply:
It removes the reliance on CAN message IDs from the application. Idea is that CAN message IDs can change but the internal message ID will always be the same for each message. Will be potentially helpful when storing in the database so we're not changing IDs often. 

To add CAN messages, you just add an ID to MESSAGE_IDS and DECODE_DATA, and then link it to a decode method (decode#) in the decode_data module. I have a cool project planned for the fall to move this stuff to a database and allow users to add messages from the GUI (and not by editing code)